README.md

```diff
@@ -1,55 +1,101 @@
+  * [Data Store Libraries](#data-store-libraries)
+  * [Projection Libraries](#projection-libraries)
+  * [Tools](#tools)
+  * [Starter Project Templates and Sample Applications](#starter-project-templates-and-sample-applications)
+* [Overview](#overview)
+* [Templates](#templates)
+* [Samples](#samples)
+* [Building](#building)
+* [Releasing](#releasing)
+* [FAQ](#faq)
+* [Acknowledgements](#acknowledgements)
+* [Further Reading](#further-reading)
+
+# Getting Started
+- If you want to start with code samples, [there's a simple `Counter` example using `Equinox.MemoryStore` for you](https://github.com/jet/equinox/blob/master/samples/Tutorial/Counter.fsx#L16)
+- If you are experienced with event sourcing, CosmosDB and F#, this sample is helpful - [~100 LOC end-to-end using CosmosDB](https://github.com/jet/equinox/blob/master/samples/Tutorial/Cosmos.fsx#L36) 
+- If you are experienced with CosmosDB and [CosmosStore](https://github.com/Dzoukr/CosmoStore), but unsure how Equinox can refer here - [Access Strategies guide](https://github.com/jet/equinox/blob/master/DOCUMENTATION.md#access-strategies)
+- For all new comers, keep reading!
+
+# Design Motivation
+Equinox's design is informed by discussions, talks and countless hours of hard and thoughtful work invested into many previous systems, [frameworks](https://github.com/NEventStore), [samples](https://github.com/thinkbeforecoding/FsUno.Prod), [forks of samples](https://github.com/bartelink/FunDomain), the outstanding continuous work of the [EventStore](https://github.com/eventstore) founders and team and the wider [DDD-CQRS-ES](https://groups.google.com/forum/#!forum/dddcqrs) community. It would be unfair to single out even a small number of people despite the immense credit that is due. Some aspects of the implementation are distilled from [`Jet.com` systems dating all the way back to 2013](http://gorodinski.com/blog/2013/02/17/domain-driven-design-with-fsharp-and-eventstore/).
+
+An event sourcing system usually has the following attributes:
+1. Storing events with good performance and debugging capabilities
+2. Transaction processing
+    - Optimistic concurrency
+    - Folding events
+3. Decoding events using codecs and formats
+4. Framework/application integration
+5. Projections
+
+Designing something that supports all of these is difficult and results in an inflexible and difficult to use framework/library; 
+thus, Equinox focuses on the core of event sourcing, items 1 and 2, so it can be as non-intrusive to your application's domain. 
+However, since the other attributes are important we've decided that they can be supported via other libraries:
+- [FSCodec](https://github.com/jet/FsCodec) supports encoding and decoding (attribute 3)  
+- [Propulsion](https://github.com/jet/propulsion) supports projections (attribute 5)
+- Since Equinox is a library and is not trying to integrate with frameworks, other wrappers libraries building on top of Equinox will support integrations (attribute 4)
```

Comment:
> Since Equinox is a library and is not trying to integrate with frameworks, other wrappers libraries building on top of Equinox will support integrations (attribute 4)

there is no "will". This needs to be called out as a non-goal. If you feel its important for there to be an answer to how to resolve it then I'd suggest:

As Equinox is a library, integrating with frameworks is very much an external concern - there's nothing stopping you or anyone else building something that e.g. wires Equinox into ASP.NET Core in some package. However, typically, managing that overlap is something that you address as part of building your application - there is no "standard ASP.NET Core Equinox Fable application template".